lib/commands/recordscreen.js

```diff
@@ -239,37 +274,34 @@ commands.startRecordingScreen = async function (options = {}) {
  *                 or the file content cannot be uploaded to the remote location.
  */
 commands.stopRecordingScreen = async function (options = {}) {
-  const {remotePath, user, pass, method} = options;
-
-  const pgrepPattern = this.isRealDevice() ? REAL_DEVICE_PGREP_PATTERN : SIMULATOR_PGREP_PATTERN;
-  const pid = await getPidUsingPattern(pgrepPattern(this.opts.device.udid));
-  let localPath = this._recentScreenRecordingPath;
-  if (_.isEmpty(pid)) {
-    log.info(`Screen recording is not running. There is nothing to stop.`);
-  } else {
-    localPath = localPath || await extractCurrentRecordingPath(pid);
-    try {
-      if (_.isEmpty(localPath)) {
-        log.errorAndThrow(`Cannot parse the path to the file created by ` +
-                          `screen recorder process from 'ps' output. ` +
-                          `Did you start screen recording before?`);
-      }
-    } finally {
-      if (!await finishScreenCapture(pid)) {
-        log.warn(`Unable to stop screen recording. Continuing anyway`);
-      }
-    }
+  const {
+    remotePath,
+    user,
+    pass,
+    method,
+  } = options;
+
+  if (!this._recentScreenRecorder) {
```

Comment:
why `recentScreenRecorder` and not just `screenRecorder`?

Reply:
I thought this name describes the purpose of the var more precisely.